### PR TITLE
Feature/eventslistredesign

### PR DESCRIPTION
**Pull request by magsyg**
Attempt at making an alternative design for events page, because it feels a litt outdated
Thought of implementing the boxes we use on the frontpage, but came up with an new design

Implemented eventquery
Would like feedback from everyone on this

first draft looks like this:

bilde

bilde

### CODE REVIEW

**Comment by magsyg** [comment]
Still needs icons for switching between old list view and imageview. 

Could be neat with 3 different views, grid view, imagerowview, and textrow view. 
Give Input.

**Comment by magsyg** [comment]
Also may look a lot like infoboxes, so may need a way to look different from them

**Comment by emilte** [comment]
Kan du legge ved et bilde av hvordan det er i dag for sammenligning?

**Comment by magsyg** [comment]
Idag:
bilde

**Comment by emilte** [comment]
Det kunne vært fint med bilde og hover effect. Utsolgte blir grå.

File: frontend/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-helmet-async": "^1.3.0",
+    "react-icons": "^4.7.1",
```
Comment (emilte) [inline frontend/package.json:38]: Vi har iconify

**Comment by magsyg** [comment]
Etter sigves endringer, så kaster jeg denne vekk

**Pull request closed without merging**